InMobi/src/main/java/com/applovin/mediation/adapters/InMobiMediationAdapter.java

```diff
@@ -159,6 +159,12 @@ public void initialize(final MaxAdapterInitializationParameters parameters, fina
 
             final JSONObject consentObject = getConsentJSONObject( parameters );
 
+            if (accountId == null) {
```

Comment:
@neelabh-parui-inmobi can we keep this condition before consentobject

Reply:
Moved up